cmake/Config.cmake

```diff
@@ -209,6 +209,10 @@ set (BOND_ENABLE_COMM
     "FALSE"
     CACHE BOOL "If FALSE, then do not build Comm")
 
+set (BOND_ENABLE_JAVA
```

Comment:
Can we make FindJavaBuildTools.cmake fail now, as they'll only be invoked if BOND_ENABLE_JAVA is true?

Reply:
Done.